lib/timeline/timeline_controller.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:async';
+
+import 'package:vm_service_lib/vm_service_lib.dart' hide TimelineEvent;
+
+import '../globals.dart';
+import 'timeline_protocol.dart';
+
+/// This class must not have direct dependencies on dart:html.
```

Comment:
top level comment for a class should decribe what it does. I would demote this to a 
// comment or add a comment indicating this class contains all timeline business logic.

Reply:
Left as /// but added a comment indicating that this class contains timeline business logic.